sdk/communication/Azure.Communication.MediaComposition.Tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Communication.MediaComposition.Models;
+
+namespace Azure.Communication.MediaComposition.Tests.samples
+{
+    /// <summary>
+    /// Basic Azure Communication.MediaComposition samples.
+    /// </summary>
+    public partial class Sample1_MediaCompositionClient
+    {
+        // TODO: Update the samples to tests when the E2E flow is working
+        private const string mediaCompositionId = "2x2Grid";
+        public async Task CreateMediaCompositionAsync()
+        {
+            var mediaCompositionClient = CreateMediaCompositionClient();
+
+            var layout = new MediaCompositionLayout()
+            {
+                Resolution = new LayoutResolution()
+                {
+                    Width = 1920,
+                    Height = 1080
+                },
+                Grid = new GridLayoutOptions(
+                    rows: 2,
+                    columns: 2,
+                    new string[2, 2] { { "jill", "jack" }, { "jane", "jerry" }
```

Comment:
Is this the right format for the 2d array or is it a nested list?

Reply:
in my unit tests I used nested lists 
![image](https://user-images.githubusercontent.com/13181430/169122880-775f4a89-5995-4024-beef-248a4c9b1fc5.png)

but I'm no c# expert

Reply:
Okay, I did the same in the Layout Component as well, so just updated accordingly,